extensions/amp-live-list/0.1/amp-live-list.js

```diff
@@ -830,6 +839,11 @@ export class AmpLiveList extends AMP.BaseElement {
   getUpdateTime() {
     return this.updateTime_;
   }
+
+  sendAmpDomUpdateEvent_() {
+    const event = new Event('amp-dom-update');
+    this.win.dispatchEvent(event);
```

Comment:
Can this be dispatched/tracked on the `this.win.document`? This will further need to be changed soon to only apply to ampdoc.